src/main/java/org/sagebionetworks/bridge/sdk/BridgeSession.java

```diff
@@ -71,11 +71,21 @@ void setSharingScope(SharingScope sharingScope) {
         this.sharingScope = sharingScope;
     }
     
+    @Override
     public DataGroups getDataGroups() {
         return dataGroups;
     }
     
     @Override
+    public Map<SubpopulationGuid,ConsentStatus> getConsentStatuses() {
+        return consentStatuses;
+    }
+    
+    void setConsentStatuses(Map<SubpopulationGuid,ConsentStatus> consentStatuses) {
+        this.consentStatuses = consentStatuses;
```

Comment:
Wrapping in an immutable map (the code that calls this sets an immutable map, but this is optimized to return the existing map).